engines/mtropolis/data.cpp

```diff
@@ -1195,6 +1230,9 @@ bool TypicalModifierHeader::load(DataReader& reader) {
 		|| !reader.readU16(lengthOfName))
 		return false;
 
+	if (loadV2Header && !reader.readU32(unknown5))
```

Comment:
Is this applying to everything using TypicalModifierHeader?  (Or at least any case of it that you've found?)  If this is global for the whole project then I think it should be flagged in DataReader instead (like the project format already is).

Reply:
Yep, it happens with all the modifiers that exist in this project so I expect it's an all-or-nothing thing and the flag could be raised up to the reader.  I'll give that a go.

Reply:
Done.